rest/blip_sync.go

```diff
@@ -94,6 +94,7 @@ func (h *handler) handleBLIPSync() error {
 		// ActiveSubprotocol only available after handshake via ServeHTTP(), so have to get go-blip to invoke callback between handshake and serving BLIP messages
 		subprotocol := blipContext.ActiveSubprotocol()
 		h.logStatus(http.StatusSwitchingProtocols, fmt.Sprintf("[%s] Upgraded to WebSocket protocol %s+%s%s", blipContext.ID, blip.WebSocketSubProtocolPrefix, subprotocol, h.formattedEffectiveUserName()))
+		base.DebugfCtx(h.ctx(), base.KeyHTTP, "BLIP Client had User-Agent: %s", base.UD(h.rq.Header.Get(base.HTTPHeaderUserAgent)))
```

Comment:
should we log this in the line above with `Updated to WebSocket` so this gets logged at info and not something that might get lost in debug?

not committed to this, but I don't know I'd look for this in a different line, like maybe https://github.com/couchbase/sync_gateway/blob/a7326093dea3d4673d857b608dc74170ef0525a6/rest/handler.go#L752

I don't have strong feelings about this if you want to drop this.